recipes/nextsilicon-cpp-subprocess/all/conanfile.py

```diff
@@ -1,27 +1,50 @@
-from conans import ConanFile, tools
+import os
+
+from conan import ConanFile
+from conan.tools.files import copy, get
+from conan.tools.layout import basic_layout
+
+required_conan_version = ">=1.52.0"
 
 
 class CppSubprocess(ConanFile):
     name = "nextsilicon-cpp-subprocess"
+    description = (
+        "Subprocessing with modern C++. "
+        "The only goal was to develop something that is as close as"
+        "Python subprocess module in dealing with processes."
+    )
     license = "MIT"
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/nextsilicon/cpp-subprocess"
-    topics = ("subprocess", "os", "fork")
-    description = ("Subprocessing with modern C++, "
-                   "The only goal was to develop something that is as close as"
-                   "python subprocess module in dealing with processes.")
-    no_copy_source = True
+    topics = ("subprocess", "os", "fork", "header-only")
+
+    package_type = "header-library"
     settings = "os", "arch", "compiler", "build_type"
+    no_copy_source = True
+
+
```

Comment:
I saw that the test_package is adding `set(CMAKE_CXX_STANDARD 11)`. I think that we should add a minimal check. Something like:

```python
    @property
    def _min_cppstd(self):
        return "11"

    def validate(self):
        if self.settings.compiler.cppstd:
            check_min_cppstd(self, self._min_cppstd)
```

Reply:
Let's not forget to add the proper import:

```python
from conan.tools.build import check_min_cppstd
```